mtls-northbound/README.md

```diff
@@ -0,0 +1,603 @@
+
+GCP's [mTLS support](https://cloud.google.com/load-balancing/docs/mtls) applies to both the [external Application Load Balancer](https://cloud.google.com/load-balancing/docs/https/setting-up-mtls-global-ext-https) and the [classic Application Load Balancer](https://cloud.google.com/load-balancing/docs/https/setting-up-mtls-ext-https), the steps are the same.
+You can [set up mutual TLS with user-provided certificates](https://cloud.google.com/load-balancing/docs/https/setting-up-mtls-ccm) or [with a private CA](https://cloud.google.com/load-balancing/docs/https/setting-up-mtls-ca-ccm).
+You can also enforce mTLS strictly or leniently, allowing missing or invalid certificates.
+In the later case, the backend is responsible for rejecting the request, in this sample, that will be the Apigee proxy.
+
+## Implementation on Apigee 
+There are no Apigee specific configuration steps, all that is required is to access the custom header values populated in the GLB configuration. The sample Apigee proxy (sample-mtls) is a simple no target proxy with an Assign Message policy that will extract the custom requst header values and return them in a JSON response. It also has a Raise Fault policy that is used to reject requests when mTLS is not strictly enforced in the GLB.
+
+## Implementation on GCP GLBs
+Adding mTLS is done on an existing GLB configuration setup for Apigee external access. 
+No changes are required to the existing certificates used for API TLS termination and the existing hostname(s) can still be used.
+
+Configuration options allow mTLS to be either enforced "strictly" or "leniently" by allowing invalid or missing certificates.
+In either case, custom header values are populated and made available to the API proxy.
+
+* Strict enforcement happens at the load balancer and invalid connections will be rejected. 
+In this case the GLB logs can be used to view connection errors for details.
+* Lenient enforcement happens in the API proxy flow. In this case, the proxy must use Conditions on the custom header values and a Raise Fault policy to reject the request.
+
+These are the high level tasks to configure mTLS on an existing GLB configuration.
+1. Create Private CA
+   1. Create Private Certificate Authority pool, “root” CA(s) and client certificates.
+   1. Create Trust Configuration for the “root” CA(s).
+   1. Create Server TLS Policy using the Trust Configuration.
+1. Update existing GLB
+   1. Update the GLB Target HTTPS Proxy to use the Server TLS Policy.
+   1. Update the GLB Backend Service to [add custom headers](https://cloud.google.com/load-balancing/docs/https/custom-headers-global#mtls-variables) for the TLS variables.
+
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro)
+2. Configure [external access](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing#external-access) for API traffic to your Apigee X instance
```

Comment:
Add a link that alternatively directs them to the `exposing-to-internet` accelerator

Reply:
OK